mercury.spec

```diff
@@ -19,6 +19,14 @@ Release: 1%{?dist}
 %global ucx 1
 %endif
 
+# do not build perf binaries on CentOS7 due to CMake PIE issues
+# see: https://cmake.org/cmake/help/latest/policy/CMP0083.html#policy:CMP0083
+%if 0%{?rhel} >= 8 || 0%{?suse_version} >= 1315
+%global __build_perf 1
+%else
+%global __build_perf 0
+%endif
```

Comment:
No need to change this on this go around, but for future reference, this would be better as an exclude list rather than include list so that new platforms being added default to the default/desired behaviour and only specifically identified platforms use the abnormal behaviour.

IOW, we should only be setting this to `0` for centos7 as a known problem platform that needs it and new platforms that get ported will get a `1` by default.

Reply:
OK yes you're right I'll make a note to change that next time